package.json

```diff
@@ -73,119 +60,96 @@
-        "cross-fetch": "^3.1.5",
-        "css-loader": "^6.7.1",
-        "cypress": "^9.7.0",
-        "cypress-dotenv": "^2.0.0",
+        "cypress": "^10.3.1",
         "del-cli": "^4.0.1",
         "dotenv": "^16.0.1",
         "eslint": "^8.16.0",
         "eslint-config-reearth": "^0.1.0",
         "eslint-plugin-cypress": "^2.12.1",
         "eslint-plugin-graphql": "^4.0.0",
-        "file-loader": "^6.1.0",
-        "hard-source-webpack-plugin": "^0.13.1",
-        "html-webpack-plugin": "^5.4.0",
-        "html-webpack-tags-plugin": "^3.0.0",
         "husky": "^8.0.1",
         "i18next-parser": "^6.4.0",
         "jsdom": "^20.0.0",
-        "json-loader": "^0.5.7",
         "lint-staged": "^12.4.2",
         "npm-run-all": "^4.1.5",
         "prettier": "^2.6.2",
-        "react-refresh": "^0.13.0",
         "read-env": "^2.0.0",
-        "serve": "^13.0.2",
-        "storycap": "^3.1.8",
-        "style-loader": "^3.3.1",
         "ts-node": "^10.8.1",
         "type-fest": "^2.12.2",
         "typescript": "^4.7.3",
         "typescript-styled-plugin": "^0.18.2",
-        "use-custom-compare": "^1.2.0",
         "vite": "^3.0.3",
         "vite-plugin-cesium": "^1.2.19",
-        "vitest": "^0.19.0",
-        "webpack": "^5.72.1",
-        "webpack-cli": "^4.9.2",
-        "webpack-dev-server": "^4.9.2",
-        "yaml-loader": "^0.8.0"
+        "vitest": "^0.19.0"
     },
     "dependencies": {
         "@apollo/client": "3.6.6",
-        "@auth0/auth0-react": "^1.10.1",
-        "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-        "@emotion/react": "^11.9.3",
-        "@emotion/styled": "^11.9.3",
-        "@monaco-editor/react": "^4.4.5",
-        "@popperjs/core": "^2.11.5",
-        "@rot1024/use-transition": "^1.0.0",
-        "@sentry/browser": "^6.19.7",
-        "@types/apollo-upload-client": "^17.0.0",
-        "@types/gapi.auth2": "^0.0.56",
-        "@types/gapi.client": "^1.0.4",
-        "@types/gapi.client.sheets": "^4.0.20201029",
-        "@types/google.picker": "^0.0.39",
-        "@types/tinycolor2": "^1.4.2",
-        "apollo-link-sentry": "^3.1.2",
-        "apollo-upload-client": "^17.0.0",
-        "array-move": "^4.0.0",
-        "axios": "^0.27.2",
-        "cesium": "^1.95.0",
+        "@auth0/auth0-react": "1.10.1",
+        "@emotion/react": "11.9.3",
+        "@emotion/styled": "11.9.3",
+        "@monaco-editor/react": "4.4.5",
+        "@popperjs/core": "2.11.5",
+        "@rot1024/use-transition": "1.0.0",
+        "@sentry/browser": "6.19.7",
+        "@types/apollo-upload-client": "17.0.0",
+        "@types/gapi.auth2": "0.0.56",
+        "@types/gapi.client": "1.0.4",
+        "@types/gapi.client.sheets": "4.0.20201029",
+        "@types/google.picker": "0.0.39",
+        "@types/tinycolor2": "1.4.2",
+        "apollo-link-sentry": "3.1.2",
+        "apollo-upload-client": "17.0.0",
+        "array-move": "4.0.0",
+        "axios": "0.27.2",
+        "cesium": "1.95.0",
```

Comment:
Probably a stupid question, but just in case: Will `yarn upgrade interactive --latest` still work same as before with us setting specific versions here?

Reply:
Also, is there anything I need to do when upgrading to avoid setting `^X.XX` accidentally

Reply:
- Of course it should work
- I found clearing yarn.lock and doing `yarn` unexpectedly installs new version of dependencies. Also renovate recommend so https://docs.renovatebot.com/configuration-options/#rangestrategy